pkg/kubelet/kubelet.go

```diff
@@ -1841,7 +1887,22 @@ func (kl *Kubelet) tryUpdateNodeStatus() error {
 		node.Status.Conditions = append(node.Status.Conditions, newCondition)
 		kl.recordNodeOnlineEvent()
 	}
+	// Get system OOM Information.
+	lastSystemOOM, timestamp, err := kl.getRecentSysOOMEvent()
```

Comment:
Isn't this going to report system OOM forever once a sytem OOM occurs? I guess this is probably okay since the upper layer will decide what to do and implement logic based on the timestamp.